### PR TITLE
Update RequestStepCategory to add Empty and Loading states for category list

### PR DESCRIPTION
**Pull request by ZhenjaHorbach**
### Details

Not here page is briefly displayed when selecting a workspace
 
### Fixed Issues

$ https://github.com/Expensify/App/issues/40999
$ https://github.com/Expensify/App/issues/41143
PROPOSAL: https://github.com/Expensify/App/issues/40999#issuecomment-2077954344

### Tests

1. Sing up with a new account
2. Create manual expense in Self DM
3. Create workspace
4. Navigate to the self DM
5. Click on 'Categorize it' under the new expense
6. Select the new workspace
7. **Verify that the loader or list of elements are shown instead of `Not here`** 
8. Open workspace setting and open categories
9. Remove all items 
10. Navigate to the self DM
11. Click on 'Categorize it'
12. Select the workspace with removed categories
13. **Verify that the component is displayed indicating an empty list** 

- [x] Verify that no errors appear in the JS console

### Offline tests

1. Sing up with a new account
2. Create manual expense in Self DM
3. Create workspace
4. Navigate to the self DM
5. Click on 'Categorize it' under the new expense
6. Select the new workspace
7. **Verify that the loader or list of elements are shown instead of `Not here`** 
8. Open workspace setting and open categories
9. Remove all items 
10. Navigate to the self DM
11. Click on 'Categorize it'
12. Select the workspace with removed categories
13. **Verify that the component is displayed indicating an empty list** 

### QA Steps

1. Sing up with a new account
2. Create manual expense in Self DM
3. Create workspace
4. Navigate to the self DM
5. Click on 'Categorize it' under the new expense
6. Select the new workspace
7. **Verify that the loader or list of elements are shown instead of `Not here`** 
8. Open workspace setting and open categories
9. Remove all items 
10. Navigate to the self DM
11. Click on 'Categorize it'
12. Select the workspace with removed categories
13. **Verify that the component is displayed indicating an empty list** 

- [x] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android: Native
    - [x] Android: mWeb Chrome
    - [x] iOS: Native
    - [x] iOS: mWeb Safari
    - [x] MacOS: Chrome / Safari
    - [x] MacOS: Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

https://github.com/Expensify/App/assets/68128028/be079aad-1cdf-4b66-85ba-561d684d2eb7

</details>

<details>
<summary>Android: mWeb Chrome</summary>

https://github.com/Expensify/App/assets/68128028/be079aad-1cdf-4b66-85ba-561d684d2eb7

</details>

<details>
<summary>iOS: Native</summary>

https://github.com/Expensify/App/assets/68128028/66aa97fb-905f-40c6-bb7f-3744e63f8ad2

</details>

<details>
<summary>iOS: mWeb Safari</summary>

https://github.com/Expensify/App/assets/68128028/a35d1918-c4f5-48b3-9f90-58a8eebf9280

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

https://github.com/Expensify/App/assets/68128028/959d4527-79ab-4568-9b9d-a87770501f05

</details>

<details>
<summary>MacOS: Desktop</summary>

https://github.com/Expensify/App/assets/68128028/a5af33fc-b225-4a82-bfb9-e9a262549b56

</details>

### CODE REVIEW

**Comment by shawnborton** [comment]
Should the empty list link to your workspace editor somehow? Otherwise it's not very helpful - it's just a dead end:
CleanShot 2024-04-30 at 12 40 58@2x

I would think we'd have a big green button at the bottom that might say "Edit categories" and it would navigate you to your workspace categories?

**Comment by dubielzyk-expensify** [comment]
I like that, though I'd suggest we should use the label `Add categories` or `Edit categories`. But maybe my English is failling here? `Edit category` seems like an odd label when you don't have any of them

**Comment by shawnborton** [comment]
Nice, I like it! And agree about making it `Edit categories` instead.

**Comment by mananjadhav** [comment]
Thanks @ZhenjaHorbach. Will finish the review by tomorrow.

File: src/pages/workspace/categories/CategoryForm.tsx [hunk 1]
```diff
@@ -28,9 +28,12 @@ type CategoryFormProps = {
 
     /** Function to validate the edited values of the form */
     validateEdit?: (values: FormOnyxValues<typeof ONYXKEYS.FORMS.WORKSPACE_CATEGORY_FORM>) => FormInputErrors<typeof ONYXKEYS.FORMS.WORKSPACE_CATEGORY_FORM>;
+
+    /** Should go back after submitting the form */
+    isShouldGoBackOnSubmit?: boolean;
```
Comment (mananjadhav) [inline src/pages/workspace/categories/CategoryForm.tsx:33]: Prefixing `should` is enough to understand it's a boolean.

```suggestion
    shouldGoBackOnSubmit?: boolean;
```

File: src/pages/workspace/categories/CategoryForm.tsx [hunk 2]
```diff
@@ -60,9 +63,13 @@ function CategoryForm({onSubmit, policyCategories, categoryName, validateEdit}:
         (values: FormOnyxValues<typeof ONYXKEYS.FORMS.WORKSPACE_CATEGORY_FORM>) => {
             onSubmit(values);
             Keyboard.dismiss();
+            if (isShouldGoBackOnSubmit) {
```
Comment (mananjadhav) [inline src/pages/workspace/categories/CategoryForm.tsx:66]: in which scenarios would we not go back?

File: src/pages/iou/request/step/IOURequestStepCategory.tsx [hunk 3]
```diff
@@ -78,7 +86,7 @@ function IOURequestStepCategory({
     const isSplitBill = iouType === CONST.IOU.TYPE.SPLIT;
     const canEditSplitBill = isSplitBill && reportAction && session?.accountID === reportAction.actorAccountID && TransactionUtils.areRequiredFieldsEmpty(transaction);
     // eslint-disable-next-line rulesdir/no-negated-variables
```
Comment (mananjadhav) [inline src/pages/iou/request/step/IOURequestStepCategory.tsx:88]: do we still need to disable this eslint rule?

Comment (mananjadhav) [reply src/pages/iou/request/step/IOURequestStepCategory.tsx:88]: I saw that but then isn't it needed for `const shouldShowEmptyState = !isLoading && !shouldShowCategory;` ?

File: src/pages/workspace/categories/CategorySettingsPage.tsx [hunk 4]
```diff
@@ -39,6 +39,7 @@ function CategorySettingsPage({route, policyCategories}: CategorySettingsPagePro
     const {translate} = useLocalize();
     const {windowWidth} = useWindowDimensions();
     const [deleteCategoryConfirmModalVisible, setDeleteCategoryConfirmModalVisible] = useState(false);
+    const backTo = route.params?.backTo ?? '';
```
Comment (mananjadhav) [inline src/pages/workspace/categories/CategorySettingsPage.tsx:42]: do we need a fallback string?

**Review by mananjadhav — COMMENTED** [review]
Just finished with the review, overall code changes look fine. I'll need a day to test this.

**Comment by mananjadhav** [comment]
@ZhenjaHorbach Can you please resolve conflicts? I'll get started on the checklist right after.

**Comment by mananjadhav** [comment]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] Android: Native
    - [ ] Android: mWeb Chrome
    - [ ] iOS: Native
    - [ ] iOS: mWeb Safari
    - [ ] MacOS: Chrome / Safari
    - [ ] MacOS: Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [x] I verified that all the inputs inside a form are aligned with each other.
    - [x] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>



</details>

<details>
<summary>Android: mWeb Chrome</summary>



</details>

<details>
<summary>iOS: Native</summary>



</details>

<details>
<summary>iOS: mWeb Safari</summary>



</details>

<details>
<summary>MacOS: Chrome / Safari</summary>



</details>

<details>
<summary>MacOS: Desktop</summary>



</details>

File: src/components/BlockingViews/BlockingView.tsx [hunk 5]
```diff
@@ -133,6 +138,7 @@ function BlockingView({
                     fill={iconColor}
                     width={iconWidth}
                     height={iconHeight}
+                    contentFit={contentFitImage}
```
Comment (mananjadhav) [inline src/components/BlockingViews/BlockingView.tsx:141]: why do we need this only for iOS?

**Review by mananjadhav — APPROVED** [review]

**Comment by mananjadhav** [comment]
The changes worked fine. @Beamanator All yours. I was a bit concerned about `includeSafeAreaPaddingBottom` being set as true, but with Button at the bottom it makes sense we don't overflow the screen.

**Comment by Beamanator** [comment]
FYI I'm adding a merge freeze HOLD which probably will come off on Monday 🙏

**Comment by mananjadhav** [comment]
@Beamanator I am assuming the merge freeze is lifted?

**Comment by Beamanator** [comment]
Sorry for the delay, y'all! Yes the merge freeze is over 👍 

@ZhenjaHorbach can you please merge main, fix the conflicts, then retest? 🙏

File: src/ROUTES.ts [hunk 6]
```diff
@@ -353,6 +353,27 @@ const ROUTES = {
...
+    },
+    SETTINGS_CATEGORY_CREATE: {
+        route: 'settings/:policyID/categories/new',
+        getRoute: (policyID: string, backTo = '') => getUrlWithBackToParam(`settings/${policyID}/categories/new`, backTo),
+    },
+    SETTINGS_CATEGORY_EDIT: {
+        route: 'settings/:policyID/categories/:categoryName/edit',
+        getRoute: (policyID: string, categoryName: string, backTo = '') =>
+            getUrlWithBackToParam(`settings/workspaces/${policyID}/categories/${encodeURIComponent(categoryName)}/edit`, backTo),
+    },
```
Comment (Beamanator) [inline src/ROUTES.ts:376]: @ZhenjaHorbach I'm not sure if I missed a convo somewhere else, but why do we need all of these new routes without `workspaces` in them? 

Also why does `SETTINGS_CATEGORY_EDIT` have `workspaces` in route passed to `getUrlWithBackToParam` but none of the other routes have that?

File: src/libs/Navigation/types.ts
Comment (Beamanator) [inline src/libs/Navigation/types.ts:1]: Is it weird we don't have any of the new `SCREENS.SETTINGS_CATEGORIES` listed in this file?

**Review by Beamanator — COMMENTED** [review]
I'm a bit confused why we're defining so many new routes, but overall the code is looking pretty decent 👍 

@mananjadhav can you please re-review?

**Comment by Beamanator** [comment]
> But since we can now use categories outside the workspace screen and not break the old logic
> I added new routes

Aah so this is the flow in your video, where you open an expense > select category > add a new category > save it > select it, right?

**Review by mananjadhav — APPROVED** [review]

**Comment by mananjadhav** [comment]
Did the retest on the Web. Worked fine.

**Review by Beamanator — DISMISSED** [review]
Looking good, seems we're still on a merge freeze though 🤷

**Comment by Beamanator** [comment]
@ZhenjaHorbach thanks for pulling main, can 1 or both of y'all please confirm this is still looking good after pulling main? 🙏  We should be able to get this merged early this week if so! (since the `freeze` is melting)

**Comment by mananjadhav** [comment]
I'll do another round of review + testing today.

**Comment by mananjadhav** [comment]
I could only test on Web yesterday. I need to test on mobile atleast once. Please give me a few hours, I'll post an update once done.

**Comment by mananjadhav** [comment]
Apologies for the delay. Tested in Web, 1 mWeb and 1 native. Attached videos for Web and Android.


https://github.com/Expensify/App/assets/3069065/b77962ed-4799-4561-a8ad-9ac8426a9b13


https://github.com/Expensify/App/assets/3069065/e658fae0-9e16-4d6f-b41a-8293687853ff


https://github.com/Expensify/App/assets/3069065/8df7880f-3564-4386-870e-0bc854a0bbc2

**Review by mananjadhav — APPROVED** [review]

File: src/pages/workspace/categories/WorkspaceCategoriesPage.tsx [hunk 7]
```diff
@@ -122,14 +123,26 @@ function WorkspaceCategoriesPage({route}: WorkspaceCategoriesPageProps) {
     );
 
     const navigateToCategorySettings = (category: PolicyOption) => {
+        if (backTo) {
+            Navigation.navigate(ROUTES.SETTINGS_CATEGORY_SETTINGS.getRoute(policyId, category.keyForList, backTo));
+            return;
+        }
         Navigation.navigate(ROUTES.WORKSPACE_CATEGORY_SETTINGS.getRoute(policyId, category.keyForList));
```
Comment (Beamanator) [inline src/pages/workspace/categories/WorkspaceCategoriesPage.tsx:130]: NAB - maybe in a follow-up we can clean this up by passing `backTo` always? so we don't have so many "if backTo exists, pass it, otherwise don't" checks? I also think it looks like `ROUTES.WORKSPACE_CATEGORY_SETTINGS.getRoute` already handles backTo existing or not, right?

**Review by Beamanator — APPROVED** [review]
LGTM! Merging cuz this has been pending for too long, but i think we should clean up the small comment i mentioned

File: src/pages/workspace/categories/WorkspaceCategoriesPage.tsx (hunk 7 above)
Comment (Beamanator) [reply src/pages/workspace/categories/WorkspaceCategoriesPage.tsx:130]: Oh geez i missed that one is SETTINGS_CATEGORY_SETTINGS, one is WORKSPACE_CATEGORY_SETTINGS. Please forgive me 😅

File: src/pages/workspace/categories/WorkspaceCategoriesPage.tsx [hunk 8]
```diff
@@ -213,35 +226,35 @@ function WorkspaceCategoriesPage({route}: WorkspaceCategoriesPageProps) {
...
                     text={translate('common.settings')}
-                    style={[isSmallScreenWidth && styles.flex1]}
+                    style={[shouldUseNarrowLayout && styles.flex1]}
                 />
             </View>
         );
     };
 
-    const isLoading = !isOffline && policyCategories === undefined;
+    const isLoading = !isOffline && policyCategories === null;
```
Comment (filip-solecki) [inline src/pages/workspace/categories/WorkspaceCategoriesPage.tsx:257]: @ZhenjaHorbach why did you change it? Right now loading state is not working properly. See [here](https://github.com/Expensify/App/pull/44137#pullrequestreview-2153445711). Reverting this change fixes the issue
